c31-swynlake/reactive-nutshell
# adding pull request template to .github

# Description

Adding a template for pull requests

Applies to issue # (n/a)

## Type of change

Please delete options that are not relevant.

# Testing Instructions

1. `git fetch --all`
2. `git checkout bm-pull-request-template`
3. check out pull_request_template.md in .github folder to make sure you agree with pull request template

# Checklist:
